generator.go

```diff
@@ -170,7 +170,10 @@ func writeCompressedFileInfo(w io.Writer, file *fileInfo, r io.Reader) error {
 		return err
 	}
 	sw := &stringWriter{Writer: w}
-	gw := gzip.NewWriter(sw)
+	gw, err := gzip.NewWriterLevel(sw, gzip.BestCompression)
+	if err != nil {
+		return err
+	}
```

Comment:
`gzip.NewWriterLevel` [documentation](http://pkg.go.dev/compress/gzip#NewWriterLevel) says:

> The compression level can be DefaultCompression, NoCompression, HuffmanOnly or any integer value between BestSpeed and BestCompression inclusive. The error returned will be nil if the level is valid.

We know `gzip.BestCompression` is a valid compression level, so the error is guaranteed to be nil. Let's simplify the code to:

```Go
gw, _ := gzip.NewWriterLevel(sw, gzip.BestCompression)
```